frontend/src/hooks/use-mobile.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from "react";
+
+const MOBILE_BREAKPOINT = 768;
+
+export function useIsMobile() {
```

Comment:
We just removed `useIsDesktop` :grin: 

Maybe we could add a comment:
`// please do not use this hook outside of shadcn components`

Reply:
Marked as deprecated :+1: